### PR TITLE
Set default period to 1s instead of 1ns

### PR DESCRIPTION
**Pull request by cottsay**

### CODE REVIEW

**Review by Karsten1987 — APPROVED** [review]
this looks good to me.
@ahendrix do you mind merging this?

File: diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp [hunk 1]
```diff
@@ -560,7 +560,7 @@ class Updater : public DiagnosticTaskVector
       node_->create_publisher<diagnostic_msgs::msg::DiagnosticArray>(
       "/diagnostics", 1);
 
-    period_ = static_cast<rcl_duration_value_t>(1.0);
+    period_ = static_cast<rcl_duration_value_t>(1000000000);  // 1.0s
```
Comment (Karsten1987) [inline diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp]: what about changing this to `1e9` for readability?

Comment (cottsay) [reply diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp]: Sure!

Comment (cottsay) [reply diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp]: Done.

**Pull request merged**